python/mleap/pyspark/__init__.py

```diff
@@ -11,3 +14,4 @@
 sys.modules['pyspark.ml'].mleap = mleap
 sys.modules['pyspark.ml'].mleap.feature = sys.modules['mleap.pyspark.feature']
 sys.modules['pyspark.ml'].mleap.feature.StringMap = StringMap
+sys.modules['pyspark.ml'].mleap.feature.MathUnary = MathUnary
```

Comment:
Small note, I'm still looking at this - if there's a better way to do it.

Reply:
One solution to this could be to subclass the JavaParams from [here](https://github.com/apache/spark/blob/3ce1dff7ba4799295a2bf89e6df35e900afc587b/python/pyspark/ml/wrapper.py#L244) and override the entire _from_java() method, but I'm not sure how everyone feels about that..